features/admin.actions.v1/components/pre-update-password-action-config-form.tsx

```diff
@@ -109,6 +116,14 @@ const PreUpdatePasswordActionConfigForm: FunctionComponent<PreUpdatePasswordActi
         mutate: mutateAction
     } = useGetActionById(actionTypeApiPath, initialValues?.id);
 
+    const {
+        data: RuleExpressionsMetaData
+    } = useGetRulesMeta(actionTypeApiPath);
+
+    // TODO: Temporary flag to show/hide the rule component.
+    const showRuleComponent: boolean = isFeatureEnabled(
+        actionsFeatureConfig, ActionsConstants.FEATURE_DICTIONARY.get("PRE_UPDATE_PASSWORD_RULE_DISABLE"));
```

Comment:
Let's keep the constant name as a feature identifier and not tie to a state.

```suggestion
        actionsFeatureConfig, ActionsConstants.FEATURE_DICTIONARY.get("PRE_UPDATE_PASSWORD_RULE"));
```